cross-compiling/README.md

```diff
@@ -27,153 +38,93 @@ The supported architectures are:
 
-## Cross-compile ROS2 SDK
+## Cross-compile ROS 2 SDK
 
-**NOTE:** there is an automated script for cross-compiling specific versions of the ROS2 SDK.
+**NOTE:** there is an automated script for cross-compiling specific versions of the ROS 2 SDK.
 For example you can run:
 
 ```
 export TARGET=raspbian
-export ROS2_DISTRO=eloquent
+export ROS2_DISTRO=foxy
 bash automatic_cross_compile.sh
 ```
 
-You can find the output in the directory `install/"$ROS2_BRANCH"_"$HEAD"_"$TARGET"`.
+You can find the output in the directory `ROS2_SDKs/"$ROS2_BRANCH"_"$HEAD"_"$TARGET"`.
 
-Let's go through an example showing how to manually cross-compile ROS2 Crystal SDK for `raspbian` board.
+Let's go through an example showing how to manually cross-compile ROS 2 Foxy SDK for `raspbian` architecture.
 
 Source the environment variables for this architecture using
 
     source env.sh raspbian
 
-If you need a sysroot for the architecture, create it.
-
-    bash get_sysroot.sh
-
+Create a sysroot for the architecture.
 This command will download a sysroot for the archicture specified by the `TARGET_ARCHITECTURE` environment variable (the argument passed to the `env.sh` script above, `raspbian` in this case).
 Note that if you already have a sysroot with the same name inside the `sysroots` directory, it will be overwritten by the new one.
 
 If you want to use your own sysroot, instead of generating a new one, you can skip the last instruction and just place your sysroot in the `sysroots` directory. Your sysroot directory must be renamed to `raspbian` or as the specified `TARGET_ARCHITECTURE` that you passed to `env.sh`.
 
-Create a ROS2 workspace and download the sources you need
+    bash get_sysroot.sh
 
-    bash get_ros2_sources.sh eloquent
+Create a ROS 2 workspace that you want to cross-compile.
+ - The cross-compilation script will mount the workspace as a Docker volume. This does not go well with symbolic links. For this reason ensure that the workspace contains the whole source code and not a symlink to the repositories.
+ - It is recommended that the workspace contains the source code of all the ROS 2 dependencies of your packages. However, you can also cross-compile individual packages if you already have the cross-compiled dependencies, see the [advanced section](advanced.md).
 
-Ignore some packages and python dependencies (note that there is a different script for each distribution you want to  cross-compile)
+We provide convenient scripts for downloading the ROS 2 sources for a specific distribution and for ignoring some packages that are not meant to be cross-compiled, e.g. visualization tools.
+If you want to cross-compile generic ROS 2 packages, see the [advanced section](advanced.md).
 
-    cd -
-    bash ignore_pkgs.sh ~/ros2_cc_ws eloquent
+    bash get_ros2_sources.sh --distro=foxy --ros2-path=~/ros2_cc_ws
```

Comment:
I think if you run this command, a directory `~` will be created in the same directory as the script `get_ros2_sources.sh`
You could do 
```
bash get_ros2_sources.sh --distro=foxy --ros2-path=$HOME/ros2_cc_ws
```

Reply:
Do you know why this happens?
`~/ros2_cc_ws` is actually the default...

Reply:
Fixed in
https://github.com/irobot-ros/ros2-performance/pull/52/commits/15b57f7e418298684be7e4df387a54c25e048f48